bitnami/redis/templates/scripts-configmap.yaml

```diff
@@ -113,11 +97,11 @@ data:
     [[ -f $REDIS_PASSWORD_FILE ]] && export REDIS_PASSWORD="$(< "${REDIS_PASSWORD_FILE}")"
     [[ -f $REDIS_MASTER_PASSWORD_FILE ]] && export REDIS_MASTER_PASSWORD="$(< "${REDIS_MASTER_PASSWORD_FILE}")"
 
-    # Waits for DNS to add this ip to the service DNS entry
-    retry_while not_exists_dns_entry
```

Comment:
Why do we need to remove this check?

Reply:
We don't need to remove this check, but I also don't see any point to check this 🤔

It makes sens to run this if we want to check this command in `if`: `getent ahosts "$HEADLESS_SERVICE" | grep -v "^${myip}"`, but I changed this and now we explicitly get info about master from sentinel.

Reply:
👍